src/test/java/com/powsybl/openloadflow/sensi/dc/DcSensitivityAnalysisTest.java

```diff
@@ -714,4 +716,30 @@ void testOldApiAdapter() {
         assertEquals(0d, values.get(2).getFunctionReference(), LoadFlowAssert.DELTA_POWER);
         assertNotNull(values.get(2).getContingencyId());
     }
+
+    @Test
+    void testDanglingLineSensi() {
```

Comment:
I am going to add a test for a `DanglingLine` injection for an AC sensitivity analysis.